src/index.css

```diff
@@ -440,6 +440,109 @@ input[type='text']:disabled {
   height: 50px;
 }
 
+.ant-table-content > table {
+  border-spacing: 0px 10px;
+  width: 100%;
+  background-color: var(--bg-color);
+  border: none !important;
+}
+.ant-table {
+  background-color: inherit !important;
+}
+
+.ant-table-body > table {
+  border-spacing: 0px 10px;
+}
+
+.ant-table-tbody > tr {
+  height: 50px;
+}
+
+.ant-table-thead {
+  background-color: #323232;
+  border-width: 1px 0px;
+  border-radius: var(--corner-radius);
+  font-size: 16px;
+  font-weight: 800;
+  height: 50px;
+}
+
+.ant-table-thead > tr > th {
+  font-size: 16px;
+  background-color: #323232 !important;
+  border-bottom: 1px solid #323232 !important;
+}
+.ant-table-thead > tr > td {
+  font-size: 16px;
+  background-color: #323232 !important;
+  border-bottom: 1px solid #323232 !important;
+}
+
+.ant-table-thead > tr > th:first-child {
+  border-left: 1px solid #323232;
+  border-top-left-radius: var(--corner-radius);
+  border-bottom-left-radius: var(--corner-radius);
+}
+
+.ant-table-thead > tr > th:last-child {
+  border-right: 1px solid #323232;
+  border-top-right-radius: var(--corner-radius);
+  border-bottom-right-radius: var(--corner-radius);
+}
+
+.ant-table-thead > tr > td:last-child {
+  border-right: 1px solid #323232;
+  border-top-right-radius: var(--corner-radius);
+  border-bottom-right-radius: var(--corner-radius);
+}
+
+.ant-table-tbody > tr > td {
+  border: 1px solid #323232;
+  border-width: 1px 0px;
+  padding: 0px 15px;
+  font-size: 16px;
+  border-bottom: 1px solid #323232 !important;
+}
+
+.ant-table-tbody > tr > td > input:focus {
+  outline: 1px solid #e0e0e0;
+  border-radius: 2px;
+}
+
+.ant-table-tbody > .active-row {
+  background-color: #757575;
+}
+
+.ant-table-tbody > tr > td:first-child {
+  border-left: 1px solid #323232;
+  border-top-left-radius: var(--corner-radius);
+  border-bottom-left-radius: var(--corner-radius);
+}
+
+.ant-table-tbody > tr > td:last-child {
+  border-right: 1px solid #323232;
+  border-top-right-radius: var(--corner-radius);
+  border-bottom-right-radius: var(--corner-radius);
+}
+
+.ant-table-placeholder {
+  color: white;
+  text-align: center;
+}
+.ant-table-cell::before {
+  background-color: inherit !important;
+}
+.ant-table-cell-ellipsis {
+  background-color: inherit;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: 150px;
+}
+.ant-table-tbody > tr.ant-table-row:hover > td {
```

Comment:
do we need a hover effect?

Reply:
yeah ant has some default hover styling that i need to override... it was making the rows white.

Reply:
it still seems to glitch when you hover. is there a way to fix that?

Reply:
fixed